lib/oli_web/controllers/api/activity_controller.ex

```diff
@@ -161,7 +161,8 @@ defmodule OliWeb.Api.ActivityController do
       {:error, {:not_authorized}} ->
         error(conn, 403, "unauthorized")
 
-      _ ->
+      e ->
+        Oli.Utils.ErrorLogger.log_error(e, "Could not create secondary activity document")
```

Comment:
This function returns a tuple with an error id and message. The message contains the error given as a parameter as well as a message that says "Contact support with issue #1234" so that the serverside error can easily be tracked down.

If we wanted to improve support here, we could capture the error message returned: `{_id, error_msg} = Oli.Utils.ErrorLogger.log_error(...` then return that in `error©nn, error_msg)`

Reply:
Sorry, I was confused as well. I thought this`Oli.Utils.ErrorLogger.log_error` function was the existing `Oli.Utils.log_error` function. Should we combine these two into a single function for consistency?

Reply:
Yes, good idea.  I just submitted a commit that does that.